src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -555,6 +555,11 @@
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
   </Node>
 
+  <Node Name="BoundDefaultPendingInference" Base="BoundExpression">
```

Comment:
>BoundDefaultPendingInference [](start = 14, length = 28)

You should be able to use the same node, whether inference has been done (i.e. it has a non-null type) or not. In other words, please handle similar to a null expression.